core/vibes/soul/sections/slideshow/index.tsx

```diff
@@ -0,0 +1,229 @@
+'use client';
+
+import { clsx } from 'clsx';
+import { EmblaCarouselType } from 'embla-carousel';
+import Autoplay from 'embla-carousel-autoplay';
+import Fade from 'embla-carousel-fade';
+import useEmblaCarousel from 'embla-carousel-react';
+import { Pause, Play } from 'lucide-react';
+import Image from 'next/image';
```

Comment:
Is that actually true @jorgemoya? This is something I think bit us in the ass for `vibes/soul-with-makeswift`. My understanding is that we want to use `BcImage` only when passing a `src` that's from BigCommerce's CDN. If the image is a local image or if it's from Makeswift or any other source, we want to make sure to use `next/image` to properly optimize it.

Now, sometimes it's impossible to know at the VIBES-component level whether the `src` will be from the BigCommerce CDN or not. For this reason, I think we should probably make a contribution to `main` so that `BcImage` conditionally uses the `loader` only when the `src` origin matches the CDN.

cc: @bookernath, @deini, @chanceaclark

Reply:
This is something I haven't explored deeply. What you suggest seems like the way to go, but wondering what others have to say.

Reply:
Sounds like we might be punting this decision so we can move forward by using `next/image` for now?